src/App.vue

```diff
@@ -40,109 +19,48 @@ const schema = {
       title: "Long Description",
       type: "string",
     },
-    done: {
-      type: "boolean",
-    },
-    dueDate: {
-      type: "string",
-      format: "date",
-      description: "The task's due date"
-    },
-    rating: {
-      type: "integer",
-      maximum: 5,
-    },
-    recurrence: {
-      type: "string",
-      enum: ["Never", "Daily", "Weekly", "Monthly"]
-    },
-    recurrenceInterval: {
-      type: "integer",
-      description: "Days until recurrence"
-    },
   },
 };
 
 const uischema = {
-  type: "HorizontalLayout",
+  type: "VerticalLayout",
   elements: [
     {
-      type: "VerticalLayout",
-      elements: [
-        {
-          type: "Control",
-          scope: "#/properties/name",
-        },
-        {
-          type: "Control",
-          scope: "#/properties/description",
-          options: {
-            multi: true,
-          }
-        },
-        {
-          type: "Control",
-          scope: "#/properties/done",
-        },
-      ],
+      type: "Control",
+      scope: "#/properties/name",
     },
     {
-      type: "VerticalLayout",
-      elements: [
-        {
-          type: "Control",
-          scope: "#/properties/dueDate",
-        },
-        {
-          type: "Control",
-          scope: "#/properties/rating",
-        },
-        {
-          type: "Control",
-          scope: "#/properties/recurrence",
-        },
-        {
-          type: "Control",
-          scope: "#/properties/recurrenceInterval",
-        },
-      ],
+      type: "Control",
+      scope: "#/properties/description",
+      options: {
+        multi: true,
+      }
     },
   ],
 };
 
-export default defineComponent({
-  name: "App",
-  components: {
-    JsonForms,
-  },
-  data() {
-    return {
-      // freeze renderers for performance gains
-      renderers: Object.freeze(renderers),
-      data: {
-        name: "Send email to Adrian",
-        description: "Confirm if you have passed the subject\nHereby ...",
-        done: true,
-        recurrence: "Daily",
-        rating: 3,
-      },
-      schema,
-      uischema,
-    };
-  },
-  methods: {
-    onChange(event: JsonFormsChangeEvent) {
-      this.data = event.data;
-    },
-  },
-  provide() {
-    return {
-      styles: myStyles,
-    };
-  },
+const data = ref({
+  name: "Send email to Adrian",
+  description: "Confirm if you have passed the subject\nHereby ...",
 });
+
+const onChange = (event: JsonFormsChangeEvent) => {
+  data.value = event.data;
+};
+
+// Provide styles to child components
+provide('styles', defaultStyles);
```

Comment:
This is not necessary if one is only using the default styles. However we would like to keep the custom styles example.